examples/server_to_qradar.py

```diff
@@ -27,19 +27,20 @@ def post_qradar(url, data_json):
 CONF.read(os.path.join(os.path.abspath(os.path.dirname(__file__)), '..', 'api.conf'))
 CLIENT = CBWApi(CONF.get('cyberwatch', 'url'), CONF.get('cyberwatch', 'api_key'), CONF.get('cyberwatch', 'secret_key'))
 
-# Retrieve the computer from Cyberwatch API
-SERVER_ID = ''
-SERVER = CLIENT.server(SERVER_ID)
+for server_item in CLIENT.servers():
+    server = CLIENT.server(str(server_item.id))
 
-# Example payload to send to qradar
+    cve_code_list = []
+    for cve in server.cve_announcements:
+        cve_code_list.append(cve.cve_code)
 
-payload = {
-    "EventCategory": "Cyberwatch Integration",
-    "Hostname": SERVER.hostname,
-    "Vulnerabilities count": SERVER.cve_announcements_count,
-}
+    # Example payload to send to qradar
+    payload = {
+        "EventCategory": "Cyberwatch Integration",
+        "Hostname": server.hostname,
+        "Status": server.status,
+        "Vulnerabilities": cve_code_list
+    }
 
-payload_json = json.dumps(payload)
-
-# Send the data to qradar
-post_qradar(CONF.get('qradar', 'url'), payload)
+    # Send the data to qradar
+    post_qradar(CONF.get('qradar', 'url'), json.dumps(payload))
```

Comment:
define :
- a method that acts like a main
- a method that takes a server and a cve_code_list and sends it to qradar that you call inside of the loop